inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/pulsar/PulsarClientService.java

```diff
@@ -246,6 +245,9 @@ public boolean sendMessage(int poolIndex, String topic, Event event,
      * @param orderEvent orderEvent
      */
     private void sendResponse(OrderEvent orderEvent) {
+        if ("false".equals(orderEvent.getHeaders().get(AttributeConstants.MESSAGE_IS_ACK))) {
+            return;
```

Comment:
How about "orderEvent.getHeaders().get(AttributeConstants.MESSAGE_IS_ACK))==null"?
It need to return directly also.

Reply:
just false need not to response message , both null ant true all need response.